app/views/govuk_publishing_components/components/_govspeak.html.erb

```diff
@@ -14,6 +14,21 @@
     <% if content.html_safe? %>
       <%= content %>
     <% else %>
+      <% puts "
```

Comment:
I've not seen puts used like this in a template before. Will this render the text inside it as written, or will it all collapse into a single chunk?

Reply:
It looks like this in the output:

```
        You've passed in unsanitised HTML into the govspeak component as the
        `content` param.

        Passing in unsafe HTML is deprecated and will be removed in a future
        version. You need to pass in a block instead or use the `capture` helper.

        See the component guide for examples.

        If you're 100% sure there's no unsanitised user input in the string you
        could also call `.html_safe` on the string or use the `raw` helper before
        passing it in.

        Called from /var/lib/jenkins/workspace/components_govspeak-content-LXCJ5KGV3W4GAGTNRFJXBGJGVEBXPWFEIRVZXHMIVRN7HGSF5OKA/app/views/govuk_publishing_components/component_guide/show.html.erb:16:in `__var_lib_jenkins_workspace_components_govspeak_content______________________________________________________app_views_govuk_publishing_components_component_guide_show_html_erb___1824013196706400841_47344039424500'
```

https://ci.integration.publishing.service.gov.uk/job/govuk_publishing_components/job/govspeak-content/2/console